src/main/scala/chisel3/iotesters/TesterOptions.scala

```diff
@@ -22,12 +22,16 @@ case class TesterOptions(
                           moreVcsFlags:    Seq[String] = Seq.empty,
                           moreVcsCFlags:   Seq[String] = Seq.empty,
                           vcsCommandEdits: String = "",
+                          moreIvlFlags:    Seq[String] = Seq.empty,
```

Comment:
Please move these new fields to the end of the options.  Adding them in the middle can create stability problems during the current development cycle.